src/core/Link/Link.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { BrowserRouter as Router, NavLink } from 'react-router-dom';
+import { useStyles } from './styles';
+
+interface LinksProps {
```

Comment:
You can also add a `target` as an optional prop, in case the link would open up on a new tab.